src/HoneybadgerLaravel.php

```diff
@@ -25,7 +25,7 @@ class HoneybadgerLaravel extends Honeybadger
         Events\JobProcessed::class,
         Events\MailSending::class,
         Events\MailSent::class,
-        Events\MessageLogged::class,
+        // Events\MessageLogged::class,
```

Comment:
```suggestion
```

Reply:
Unless there's a reason this should remain as a comment, I think it should be removed.

Reply:
Thanks, it should be removed.